R/post-body.R

```diff
@@ -1,7 +1,7 @@
 postBodyFilter <- function(req){
   handled <- req$.internal$postBodyHandled
   if (is.null(handled) || handled != TRUE){
-    body <- req$rook.input$read_lines()
+    body <- paste0(req$rook.input$read_lines(), collapse = "\n")
```

Comment:
What if post body has valid returns in it?  Such as a csv, or a text field that has intentional returns in it?  Blanket removing returns isn't the correct solution.

Reply:
`read_lines()` should only be returning a character vector for each line of text.  If the text has a newline value in it, it should be escaped and be interpreted as regular text.  

By collapsing the text with `\n`, the original new line locations are preserved.  This makes the postBody a single character value (correct), not a vector of character values (incorrect).